pkg/build/util/util.go

```diff
@@ -32,6 +32,10 @@ const (
 
 	// buildPodSuffix is the suffix used to append to a build pod name given a build name
 	buildPodSuffix = "build"
+
+	// BuildCacheBlobs is the directory used to store a cache for the blobs to be
+	// reused across builds.
+	BuildBlobsMetaCache = "/var/cache/blobs"
```

Comment:
1) godoc needs to be fixed
2) i really meant change the directory name, e.g "/var/cache/blobsmeta" or "/var/cache/blobs/meta"

Reply:
oh, not sure then, @mrunalp ideas? I'm ok with either

Reply:
Lets go with `/var/cache/blobs/meta`.

Reply:
@runcom if you update this i think we can put it in the merge queue.

Reply:
done and re-pushed

Reply:
Is this the blob info cache (which is stored under `/var/lib/containers/cache` by default), or the blob cache?  If it's the latter, it might make sense to set `$BUILD_BLOBCACHE_DIR` to the location of the volume mount.